src/ui/atoms/inputBase/InputBase.tsx

```diff
@@ -55,6 +55,10 @@ export type InputBaseProps = {
    * If true and multiline, the user can't resize the text area.
    */
   readonly disableAreaResize?: boolean
+  /**
+   * If true, the user can only resize the height of the text area.
+   */
+  readonly resizeHeightOnly?: boolean
```

Comment:
I'm a little embarrassed about using this property.

Indeed, if I understand correctly, the purpose of this property is to prevent the resize of the text area when it is in full-width.

What bothers me is exposing a property to the user which, in the end, is only used for internal modifications to the component.

I think it should just be a condition carried by the `InputBase` component and attached to the `fullWidth` and `multiline` properties.

Thus, it shouldn't be exposed as `props`.